package.json

```diff
@@ -19,6 +19,8 @@
   },
   "homepage": "https://github.com/hrr43fec-luke/andrew-proxy#readme",
   "dependencies": {
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "node-fetch": "^2.6.0",
+    "nodemon": "^2.0.2"
```

Comment:
technically should be a dev dependency? As I'm deploying to AWS, I see you can npm install only production dependencies since you won't be building on AWS.

Reply:
This is good to know. I'll change them. Thanks!